sycl/include/sycl/ext/intel/experimental/esimd/detail/memory_intrin.hpp

```diff
@@ -16,6 +16,8 @@
 #include <sycl/ext/intel/esimd/detail/defines_elementary.hpp>
 #include <sycl/ext/intel/esimd/detail/memory_intrin.hpp>
 
+using uint = unsigned int;
```

Comment:
I don't think spoiling global namespace like this is good. Is there a standard header where it is defined?

Reply:
+1. I think this needs to be fixed

Reply:
Oh, this is already fixed.